### PR TITLE
feat: Allow specifying extraMasks for URL normalizer

### PR DESCRIPTION
**Pull request by AeroNotix**

### CODE REVIEW

File: src/normalizers.js [hunk 1]
```diff
@@ -16,12 +14,15 @@ const urlParser = new UrlValueParser();
...
+ * @param {Array} extraMasks - array of regexps which will replace
+ * values in the URL.
  * @param {string} [placeholder='#val'] - the placeholder that will replace id
  * like params in the url path.
  * @returns {string} a normalized path, withoud ids.
  */
-function normalizePath (originalUrl, placeholder = '#val') {
+function normalizePath (originalUrl, extraMasks = [], placeholder = '#val') {
   const { pathname } = url.parse(originalUrl);
+  const urlParser = new UrlValueParser({extraMasks: extraMasks});
```
Comment (joao-fontenele) [inline src/normalizers.js]: I know I haven't configured linting properly, but code style is:

```js
const urlParser = new UrlValueParser({ extraMasks });
```

**Comment by AeroNotix** [comment]
@joao-fontenele hi - made the requested changes.

File: README.md [hunk 2]
```diff
@@ -50,6 +50,11 @@ app.use(promMid({
    * curl -X GET user:password@localhost:9091/metrics
    */
   // authenticate: req => req.headers.authorization === 'Basic dXNlcjpwYXNzd29yZA==',
+  /**
+   * Uncommenting the `extraMasks` callback will use the list of regexes to
```
Comment (joao-fontenele) [inline README.md]: better wording would be: `Uncommenting the `extraMasks` config`

**Comment by AeroNotix** [comment]
rebased w/ master and squashed

**Pull request merged**